dasharo-stability/capsule-update.robot

```diff
@@ -248,26 +270,46 @@ Upload Required Files
-
-    # Move the directory to ESP partition so the tests work even if root
-    # file-system is part of LVM
-    Execute Command In Terminal    rm -r /boot/efi/capsule_testing
-    Execute Command In Terminal    mv /capsule_testing /boot/efi
+
+    IF    ${TESTS_IN_UBUNTU_SUPPORT}
+        Go To Ubuntu Prompt
+        # Send File To DUT uses regular user, so prepare target directory in as root
+        Execute Command In Terminal    rm -r /capsule_testing
+        Execute Command In Terminal    mkdir /capsule_testing
+        Execute Command In Terminal    chmod 777 /capsule_testing
+        Log To Console    Sending ./dasharo-stability/capsule-update-files/CapsuleApp.efi
+        Send File To DUT    ./dasharo-stability/capsule-update-files/CapsuleApp.efi    /capsule_testing/CapsuleApp.efi
+        Log To Console    Sending ${CAPSULE_FW_FILE}
+        Send File To DUT    ${CAPSULE_FW_FILE}    /capsule_testing/valid_capsule.cap
+        Log To Console    Sending ./dl-cache/edk2/${file_name}_wrong_cert.cap
+        Send File To DUT    ./dl-cache/edk2/${file_name}_wrong_cert.cap    /capsule_testing/wrong_cert.cap
+        Log To Console    Sending ./dl-cache/edk2/${file_name}_invalid_guid.cap
+        Send File To DUT    ./dl-cache/edk2/${file_name}_invalid_guid.cap    /capsule_testing/invalid_guid.cap
+        # Move the directory to ESP partition so the tests work even if root
+        # file-system is part of LVM
+        Execute Command In Terminal    rm -r /boot/efi/capsule_testing
+        Execute Command In Terminal    mv /capsule_testing /boot/efi
+    ELSE IF    ${TESTS_IN_WINDOWS_SUPPORT}
+        Go To Windows Prompt
+        Log To Console    Sending ./dasharo-stability/capsule-update-files/CapsuleApp.efi
+        SSHLibrary.Put File
+        ...    ./dasharo-stability/capsule-update-files/CapsuleApp.efi
+        ...    C:\\capsule_testing\\CapsuleApp.efi
+        Log To Console    Sending ${CAPSULE_FW_FILE}
+        SSHLibrary.Put File    ${CAPSULE_FW_FILE}    C:\\capsule_testing\\valid_capsule.cap
+        Log To Console    Sending ./dl-cache/edk2/${file_name}_wrong_cert.cap
+        SSHLibrary.Put File    ./dl-cache/edk2/${file_name}_wrong_cert.cap    C:\\capsule_testing\\wrong_cert.cap
+        Log To Console    Sending ./dl-cache/edk2/${file_name}_invalid_guid.cap
+        SSHLibrary.Put File    ./dl-cache/edk2/${file_name}_invalid_guid.cap    C:\\capsule_testing\\invalid_guid.cap
+        Execute Command In Terminal    mountvol b: /s
+        Set Prompt For Terminal    PS B:\\>
+        Execute Command In Terminal    b:
+        Execute Command In Terminal    rmdir /q .\\capsule_testing\\
+        Execute Command In Terminal    mkdir capsule_testing
+        Execute Command In Terminal    copy C:\\capsule_testing\\*.* B:\\capsule_testing
+    ELSE
+        Fail    No Ubuntu nor Windows support.
+    END
 
     # Make sure file-system data is pushed to disks before resetting a platform
     Execute Command In Terminal    sync
```

Comment:
`sync` is Linux command, there is no reason to try running it on Windows:

```
>> sync
>> 
sync : The term 'sync' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ sync
+ ~~~~
    + CategoryInfo          : ObjectNotFound: (sync:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

Instead, under Windows you should use `mountvol b: /d`, I think it would make sure that the caches are flushed.

Reply:
Changed: d1fc45aebebd8edb92cf928b184fc6276b6ea9e7